tests/test_other.py

```diff
@@ -9335,7 +9335,10 @@ def test_proxy_to_pthread_stack(self):
     # DEFAULT_PTHREAD_STACK_SIZE.
     self.do_smart_test(test_file('other/test_proxy_to_pthread_stack.c'),
                        ['success'],
-                       emcc_args=['-s', 'USE_PTHREADS', '-s', 'PROXY_TO_PTHREAD', '-s', 'DEFAULT_PTHREAD_STACK_SIZE=64kb', '-s', 'TOTAL_STACK=128kb', '-s', 'EXIT_RUNTIME'])
+                       emcc_args=['-sUSE_PTHREADS', '-sPROXY_TO_PTHREAD',
+                                  '-sDEFAULT_PTHREAD_STACK_SIZE=64kb',
+                                  '-sTOTAL_STACK=128kb', '-sEXIT_RUNTIME',
+                                  '--profiling-funcs'])
```

Comment:
was `--profiling-funcs` left over from debugging?

Reply:
Yes, I find it gives much better stack traces.  Would you rather I remove it?

Reply:
I'm not opposed to it, but isn't that true of all tests?

If there's a specific reason it helps here more than other tests then sgtm. Otherwise I think it's confusing as it suggests that test interacts with that option somehow. (But I guess a comment could fix that.)